### ISSUE TITLE
Add Maintainers Sponsor Links

### ISSUE THREAD

**Issue by hollowaykeanho** [hollowaykeanho/Upscaler#40]
# Description
```
Please provide a short description of what you have encountered below.
```
Let's get the business part of things up at least for me and Joly0. Jean and Cory stated they do not wish to participate.

# Link
```
Please provide the documentation link(s) below.
```
README.md

# Expected Presentation
```
Please specify the expected presentation of the documentation.
```
Each maintainer has a "sponsor" button next to their listing, allowing the public to sponsor him/her.

# Current Presentation
```
Please specify the current presentation of the documentation.
```
no sponor button is available for any maintainer.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by JeanShuralyov** [comment]
I'm out. Please proceed.

**Comment by hollowaykeanho** [comment]
noted

**Comment by Joly0** [comment]
Btw, i have requested a sponsor page on github and got accepted today

**Comment by hollowaykeanho** [comment]
Congratulations! =)

**Comment by hollowaykeanho** [comment]
Implemented in 51aad4232ce87afc83d8f133858f5ad602d7f095
